include/nlohmann/detail/conversions/from_json.hpp

```diff
@@ -139,6 +139,7 @@ inline void from_json(const BasicJsonType& j, typename BasicJsonType::number_int
     get_arithmetic_value(j, val);
 }
 
+#if JSON_NO_ENUM
```

Comment:
Shouldn't this be `#if !JSON_NO_ENUM`?

Reply:
Yes. Changing from `#ifdef` reversed the logic.